### ISSUE TITLE
Disabled state for tinymce-angular editor don't works correctly

### ISSUE THREAD

**Issue by protekshen** [tinymce/tinymce-angular#33]
When component <editor> get disabled state to then elements inside TinyMCE are disabled incorrect (Users don't understand that editor component is lock/disabled). In the disabled state we expect the inactive fade component, but now it's not so.

We have 2 problems in TinyMCE:
1. Text (description) has bright black color , although the component is disable;
2. Buttons of editor can be get focus/hover on the element,  although the component is disable;

![image](https://user-images.githubusercontent.com/13870417/44842198-e76deb80-ac4d-11e8-9ef9-10784f801807.png)

Please, fix this behavors for <editor disabled>

**Comment by fyrkant** [comment]
How do you set the disabled state?

**Comment by protekshen** [comment]
@fyrkant add 'disabled' to <editor> tag (Example: <editor disabled>)

![image](https://user-images.githubusercontent.com/13870417/44977168-f61b1200-af6f-11e8-9d0e-04ccf95f2049.png)

**Comment by fyrkant** [comment]
That property is not set up to change the mode on the TinyMCE Editor, so that won't work right now I'm afraid. I'll change this to a feature request.

**Comment by protekshen** [comment]
@fyrkant I understood correctly? You will fix it?

**Comment by protekshen** [comment]
@fyrkant Thank you very much, we will wait.
Have a good day ! :)

**Comment by protekshen** [comment]
@fyrkant Hello, what about this? there are successes?

**Comment by fyrkant** [comment]
@protekshen 

Sorry I have not had any time to implement this yet and I can't really give an estimated time when I will have time.

**Comment by fyrkant** [comment]
Are you sure you have updated to the latest version?

**Comment by protekshen** [comment]
Yes, I was delete my node_modules folder and then update my package.json to "    "@tinymce/tinymce-angular": "2.2.0"," and then did npm install

**Comment by protekshen** [comment]
yes

```
<editor #editor
        [apiKey]="key"
        [init]="init"
        [(ngModel)]="text"
        [disabled]="disabled"
        (ngModelChange)="onModelChange($event)">
</editor>
```

**Comment by fyrkant** [comment]
And disabled is a boolean variable on your class? Can you create an example app that I can click around in maybe, that might be faster for me to see what's wrong.
